test/boost/user_types_test.cc

```diff
@@ -111,16 +111,18 @@ SEASTAR_TEST_CASE(test_invalid_user_type_statements) {
 
         // non-frozen UDTs can't be inside collections, in create table statements...
         REQUIRE_INVALID(e, "create table bad (a int primary key, b list<ut1>)",
-                "Non-frozen user types or collections are not allowed inside collections: list<ut1>");
+                "Non-frozen user types or collections are not allowed inside collections: list<ks.ut1>");
```

Comment:
So your change affected how the UDTs are rendered inside error messages (now they are written fully qualified)?
But why? Is it because of some side effects caused by the calls to `prepare_internal(..., user_types)` inside `do_prepare`?

It's kinda surprising - it was not the intention of the change. But it seems like a harmless addition. Anyway, it should be mentioned and explained in the commit message.